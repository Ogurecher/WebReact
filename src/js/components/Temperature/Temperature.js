import React, { Component } from 'react';

export default class Temperature extends Component {

    render() {
        return (
            <p> {this.props.weather.main.temp}Â°C </p>
        );
    }
}