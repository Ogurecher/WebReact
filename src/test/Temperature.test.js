import React from 'react';
import renderer from 'react-test-renderer';

import Temperature from '../js/components/Temperature/Temperature';

describe('Temperature', function() {
    test('snapshot renders', function() {
        const weatherStub = {
            main: {
                temp: '1'
            }
        };

        const component = renderer.create(<Temperature weather={weatherStub} />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});