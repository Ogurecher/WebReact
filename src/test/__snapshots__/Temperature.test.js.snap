// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Temperature snapshot renders 1`] = `
<p>
   
  1
  Â°C 
</p>
`;
